tests/core/test_trackers.py

```diff
@@ -566,6 +566,21 @@ def test_last_executed_has_not_name():
     assert tracker.last_executed_action_has("another") is False
 
 
+def test_events_metadata():
```

Comment:
is this not covered by `test_event_metadata_dict` ?

Reply:
They cover almost the same things, but I wanted to make sure that the metadata is returned when serializing the entire Tracker as well (instead of just the Event).